tests/test_sign_tx.py

```diff
@@ -0,0 +1,141 @@
+import pytest
+import rlp
+from hashlib import blake2b
+from application_client.transaction import Transaction
+from application_client.command_sender import CommandSender, Errors
+from application_client.utils import create_ae_curve_path
+from application_client.response_unpacker import unpack_sign_response
+from ragger.bip import calculate_public_key_and_chaincode, CurveChoice
+from utils import check_signature_validity
+from ragger.error import ExceptionRAPDU
+
+
+def test_sign_tx_short_tx(backend, scenario_navigator):
+    tx = Transaction(
+        tag=0x0C,
+        vsn=0x01,
+        sender=b'\x01\xf7^S\xf5x""zX\xb4c\t]m\xabe|\xab\x80Et\xbeb\xde\x0b\xe1\xf9Ry\xd0\x907',
+        recipient=b'\x01\xf7^S\xf5x""zX\xb4c\t]m\xabe|\xab\x80Et\xbeb\xde\x0b\xe1\xf9Ry\xd0\x907',
+        amount=0x1111D67BB1BB0000,
+        fee=0x0F4C36200800,
+        ttl=0x00,
+        nonce=0x0A,
+        payload=b"Lorem ipsum dolor sit amet",
+    )
```

Comment:
So far looks good, but consider reusing js sdk or something (node's debug endpoints on testnet?) to build transactions